gapic/src/main/com/google/gapid/widgets/LoadableImage.java

```diff
@@ -98,6 +104,18 @@ protected void onUiThreadError(Void error) {
     return this;
   }
 
+  public LoadableImage unload() {
```

Comment:
```
if (--loadCount != 0 || state != State.LOADING) {
  return this;
}
future.cancel(true);
updateImage(null);
state = State.NOT_STARTED;
return this;
```
That way, once an image has finished loading, it won't be re-loaded again, after going offscreen.